client/scss/application.scss

```diff
@@ -22,12 +22,13 @@
 @import 'grids/grid_size';
 
 // Components
-@import 'components/skip_link';
 @import 'components/block_description';
 @import 'components/blockquote';
 @import 'components/body_content';
+@import 'components/buttons';
```

Comment:
I jiggled these around to keep the components list alphabetical – that way as and when we have a longer list, there's less likelihood of accidentally duplicating items in the manifest.